docker-bench-security.sh

```diff
@@ -103,15 +101,27 @@ main () {
       benchcont="$c"
     fi
   done
+  
```

Comment:
White spaces.

Reply:
Fixed in 3d02432bc8255c3b115ba335d0fe4c3e24a8cd37